src/lib/pages/backoffice/Document/DocumentDetails/DocumentRelations/RelationLanguages/RelationLanguagesModal/RelationLanguagesModal.js

```diff
@@ -47,6 +47,8 @@ export default class RelationLanguagesModal extends Component {
     return (
       <RelationModal
         triggerButtonContent="Add language relations"
+        disabled={!documentDetails.metadata.languages}
```

Comment:
could you do the same for the edition relation? I don't see it in the PR

Reply:
I did it here https://github.com/inveniosoftware/react-invenio-app-ils/blob/f367d5f36a56ed2fe59d7e9b58dc320379b63a76/src/lib/modules/Relations/backoffice/RelationEdition/RelationEditionModal/RelationEditionModal.js#L70